### PR TITLE
updated troubleshooting guide in case redis-server is already running outside of docker

### PR DESCRIPTION
**Pull request by jecdohmann**
updated trouble shooting markdown if redis-server fails

### CODE REVIEW

File: TROUBLESHOOTING.md [hunk 1]
```diff
@@ -2,18 +2,19 @@
 
 ## Issues starting MySQL or PostgreSQL with docker-compose
```
Comment (chewselene) [inline TROUBLESHOOTING.md]: Title should be updated as well

File: TROUBLESHOOTING.md [hunk 2]
```diff
@@ -2,18 +2,19 @@
...
 
-If you have any trouble starting the MySQL/PostgreSQL database, make sure any database service is not already
-running outside of docker. On OSX, you can stop the MySQL and PostgreSQL services by executing:
+If you have any trouble starting the MySQL/PostgreSQL database or the redis server, make sure any database service is not already
```
Comment (pmantica1) [inline TROUBLESHOOTING.md]: Typo: redis server is not a database service.

File: TROUBLESHOOTING.md [hunk 3]
```diff
@@ -1,19 +1,20 @@
...
 brew services stop postgresql
+brew services stop redis-server
 ```
 or on Ubuntu with:
 ```bash
 service mysql stop
 service postgresql stop
+service redis-server stop
 ```
-
```
Comment (pmantica1) [inline TROUBLESHOOTING.md]: nit: Do not delete this line. We want to leave one space before each new heading. (Also in general, please do not submit style changes that are not relevant to the PR).

**Comment by pmantica1** [comment]
Do you mind fixing the title of this PR?

**Pull request merged**